src/sdk/bots-brain/BotsStore.ts

```
@@ -862,43 +879,64 @@ export class BotsStore {
-					missing: missingTables,
-				});
-				await this.initializeSchema();
-			}
-			else {
-				LogEngine.info('Database schema verified', {
-					tablesFound: foundTables,
-					botsBrainReady: foundTables.includes('storage_cache'),
-				});
+			try {
+				await Promise.race([this.performSchemaCheck(), timeoutPromise]);
+			} finally {
```

ESLint

[error] Closing curly brace appears on the same line as the subsequent block.

(brace-style)